testing/bdd/features/timing/ebuttd_resolved_timings_on_elements.feature

```diff
@@ -23,7 +23,27 @@ Feature: Resolving timings on elements
       | p_begin  | p_end    | span1_begin | span1_end | span1_resulted_begin_time | span1_resulted_end_time |
       | 00:00:05 |          | 00:00:02    | 00:00:08  | 00:00:07                  | 00:00:13                |
       | 00:00:10 | 00:00:15 | 00:00:01    | 00:00:02  | 00:00:11                  | 00:00:12                |
-      | 00:00:10 | 00:00:13 |             | 00:00:04  | 00:00:10                  | 00:00:13                |
+      | 00:00:10 | 00:00:15 |             | 00:00:04  | 00:00:10                  | 00:00:14                |
+
+
+  Scenario: Timings present on both p and span elements resulting in p times
+    Given an xml file <xml_file>
+    When  it has p begin time <p_begin>
+    And   it has p end time <p_end>
+    And   it has span1 begin time <span1_begin>
+    And   it has span1 end time <span1_end>
+    When  the document is generated
+    And   the EBU-TT-Live document is denested
+    And   the EBU-TT-Live document is converted to EBU-TT-D
+    Then  EBUTTD document is valid
+    And   span1 resulted begin time is <span1_resulted_begin_time>
+    And   span1 resulted end time is <span1_resulted_end_time>
+    And   timings present on p
+
+    Examples:
+      | p_begin  | p_end    | span1_begin | span1_end | span1_resulted_begin_time | span1_resulted_end_time |
+      | 00:00:10 | 00:00:13 |             | 00:00:04  | None                      | None                    |
+      | 00:00:10 | 00:00:13 | 00:00:00    | 00:00:04  | None                      | None                    |
```

Comment:
I'm not convinced this is working. Your example on line 45 almost matches line 26. Why does one produce span timings but not the other?

Reply:
This is super subtle, but they are genuinely different. You're right that they look similar, but the differences have a significant impact.

Line 26 has a span that ends before its parent p. Line 45 has one that is truncated by the end time of its parent p.

That means that on line 26 the span times are different to the parent p times, and the rule in that case is that the times move down to the spans.
But on line 45 the span times end up being the same as the parent p times, and the rule in that case is that we put the times on the p.